website/docs/reference/warehouse-setups/vertica-setup.md

```diff
@@ -2,22 +2,29 @@
 title: "Vertica setup"
 id: "vertica-setup"
 meta:
-  maintained_by: Community
-  authors: Matthew Carter, Andy Regan, Andrew Hedengren
-  github_repo: 'mpcarter/dbt-vertica'
+  maintained_by: Vertica
+  authors: Vertica (Former authors:- Matthew Carter, Andy Regan, Andrew Hedengren)
+  github_repo: 'https://github.com/vertica/dbt-vertica/'
   pypi_package: 'dbt-vertica'
-  min_core_version: 'v0.21.0'
+  min_core_version: 'v1.3.0'
   cloud_support: Not Supported
-  min_supported_version: 'Vertica 10.0'
-  slack_channel_name: 'n/a'
-  slack_channel_link: 'https://www.getdbt.com/community/'
+  min_supported_version: 'Vertica v12.0'
+  slack_channel_name: '#adapter-ecosystem'
+  slack_channel_link: 'https://app.slack.com/client/T0VLPD22H/C030A0UF5LM/thread/C030A0UF5LM-1669649490.022339'
```

Comment:
this should stay the same. feel free to request a #db-vertica channel in the #channel-requests channel!
```suggestion
  slack_channel_name: 'n/a'
  slack_channel_link: 'https://www.getdbt.com/community/'
```